src/Autowhatever.js

```diff
@@ -30,6 +30,10 @@ export default class Autowhatever extends Component {
     shouldRenderSection: PropTypes.func,   // This function gets a section and returns whether it should be rendered, or not.
     renderSectionTitle: PropTypes.func,    // This function gets a section and renders its title.
     getSectionItems: PropTypes.func,       // This function gets a section and returns its items, which will be passed into `renderItem` for rendering.
+    inputElement: PropTypes.oneOfType([    // This element is used to render the input
+      PropTypes.func,
+      PropTypes.string
+    ]),
```

Comment:
I tried PropTypes.node but it did not work neither for Components, nor stateless functions nor strings. So I ended up with func + string which covers all cases.